pkg/agent/kube.go

```diff
@@ -41,6 +41,11 @@ import (
 
 const AnnotationGitRepository = "gimlet.io/git-repository"
 const AnnotationGitSha = "gimlet.io/git-sha"
+const AnnotationDocsLink = "opensca.dev/documentation"
```

Comment:
Please use this format:
```
opensca.dev_v0.1.0/version.sha
opensca.dev_v0.1.0/version.link
opensca.dev_v0.1.0/description
```

I will update opensca.dev specification accordingly.